controllers/recipes.js

```diff
@@ -155,8 +155,17 @@ module.exports = {
   },
   searchRecipe: async (req, res) => {
     console.log("🔎 The search button is working.");
+    console.log("Query params:", req.query);
+    const currentPage = "searchResults";
+    const skip =
```

Comment:
For increased readability this could be refactored to

`const skip = Math.max(0, parseInt(req.query.skip || "0", 10));`

Reply:
Aside: if you decide to do the separate refactoring branch for the `limit` var, then refactoring this `skip` evaluation in other methods could be a part of that.

Reply:
I like this! and that we are making it global as well